# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/EditorUtils.h
# Commit: 71670cedaf26
# Full Hash: 71670cedaf26214f3bcb8562a445cf1ef24f4db6
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2021-06-30 09:36:48
# Regressor Bug: 1714914
# File Overlap Count: 1
# Description:
#   Bug 1714914 - Make editor classes treat offset in parent node as uint32_t r=m_kato
#   
#   It should be treated as `uint32_t` since DOM API does so.  However, there are
#   some exceptions:
#   
# ==============================================================================

diff -r f5dcc2d793d6 -r 71670cedaf26 editor/libeditor/EditorUtils.h
--- a/editor/libeditor/EditorUtils.h	Tue Jun 29 22:00:09 2021 +0000
+++ b/editor/libeditor/EditorUtils.h	Wed Jun 30 07:07:28 2021 +0000
@@ -1219,7 +1219,7 @@
    * Returns true if aSelection includes the point in aParentContent.
    */
   static bool IsPointInSelection(const Selection& aSelection,
-                                 const nsINode& aParentNode, int32_t aOffset);
+                                 const nsINode& aParentNode, uint32_t aOffset);
 
   /**
    * Create an nsITransferable instance which has kUnicodeMime and